Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -118,38 +118,120 @@ def adjoint(self,x, out=None):
         raise NotImplementedError
     
     @staticmethod
-    def PowerMethod(operator, iterations, x_init=None):
-        '''Power method to calculate iteratively the Lipschitz constant
+    def PowerMethod(operator, iterations=10, tolerance = 1e-5, initial=None, verbose=False):

```

Comment:
as we use random, do we want to add `seed`? Or this can be set outside this function?

Reply:
Suggested new signature not to break backward compatibility:

```python

def PowerMethod(operator, iterations=10, initial=None, tolerance=1e-5, verbose=False, x_init=None):
```
So if `x_init` is passed positionally it will still work, if it's passed with keyword it will be picked. 
We need to add logic to check `if initial is None and x_init is not None`  raise a warning that `x_init` is deprecated and make sure `initial=x_init`. If both are passed `x_init` should be neglected and a warning raised.

Reply:
Fixed